domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ClaimDocumentType.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+public enum ClaimDocumentType {
```

Comment:
How about renaming this to `DocumentType`?

Reply:
I thought of the same but DocumentType is generic and may be confused with mimetype . so to be specific I used this.